src/foam/nanos/crunch/UserCapabilityJunctionRefine.js

```diff
@@ -127,8 +127,8 @@ foam.CLASS({
       javaFactory: `
         var payload = new CapabilityJunctionPayload();
         // Temporary so outdated test journals work
-        if ( statusIsSet_ ) payload.setStatus(status_);
-        if ( dataIsSet_ ) payload.setData(data_);
+        if ( statusIsSet_ ) payload.setStatus(getStatus());
+        if ( dataIsSet_ ) payload.setData(getData());
```

Comment:
what is the point of these two? getData() calls  getPayload().getData() 
or is there something else going on?
data and status are storageTransient, so any old journals with 'data' and 'status' properties will be ignored.

Reply:
need @KernelDeimos  to comment please, this was the code before the fix made here https://github.com/foam-framework/foam2/pull/4745, and since that fix was specifically for recursion caused by checking dataIsSet, I reverted it as well

Reply:
This does cause a recursion issue, so `status_` and `data_` is still better. I'm assuming when outdated UCJs are read from the journal the proxying getter and setter aren't called, otherwise this can be removed. Or, if we don't have any outdated UCJs, this can definitely be removed.

Reply:
reverted revert of fix